src/Log/Log.php

```diff
@@ -271,11 +272,11 @@ public static function levels(): array
      * ```
      *
      * @param array|string $key The name of the logger config, or an array of multiple configs.
-     * @param array|null $config An array of name => config data for adapter.
+     * @param \Psr\Log\LoggerInterface|\Closure|array|null $config An array of name => config data for adapter.
      * @return void
      * @throws \BadMethodCallException When trying to modify an existing config.
      */
-    public static function setConfig($key, $config = null): void
+    public static function setConfig(array|string $key, LoggerInterface|Closure|array|null $config = null): void
```

Comment:
I think `LoggerInterface` is the correct base class here. Wasn't sure if it had to be a `BaseLog` instance, but I don't think so.